frontend/packages/client/src/components/ProposalInformation/ProposalInformation.js

```diff
@@ -0,0 +1,177 @@
+import { useEffect, useRef, useState } from 'react';
+import {
+  useCommunityDetails,
+  useVotingResults,
+  useWindowDimensions,
+} from 'hooks';
+import Information from './Information';
+import ResultsPanel from './ResultsPanel';
+
+const ProposalInformation = ({
+  creatorAddr = '',
+  strategyName = '',
+  isCoreCreator = false,
+  ipfs = '',
+  ipfsUrl = '',
+  startTime = '',
+  endTime = '',
+  proposalId = '',
+  computedStatus,
+  communityId,
+  proposalStrategy,
+  proposalMaxWeight,
+  proposalMinBalance,
+  openStrategyModal = () => {},
+}) => {
+  // stores navbar height calculated after component is mounted
+  const [navbarHeight, setNavbarHeight] = useState(0);
+
+  useEffect(() => {
+    setNavbarHeight(document.querySelector('header').offsetHeight);
+  }, []);
+
+  const { height: windowHeight, width: windowWidth } = useWindowDimensions();
+
+  // stores when user scrolls
+  const [scroll, setScroll] = useState(0);
+
+  // stores style to apply when info bar needs to be fixed
+  const [fixedStyle, setFixedStyle] = useState(null);
+  // ref of the panel info component
+  const ref = useRef(null);
+  // ref of the parent component
+  const parentRef = useRef(null);
+  // used to store return point
+  const topRef = useRef({ pointStatic: null });
+
+  const { data: votingResults } = useVotingResults(proposalId);
+
+  const { data: community } = useCommunityDetails(communityId);
+  const {
+    name: communityName,
+    logo: communityLogo,
+    slug: communitySlug,
+    strategies: commnunityStrategies,
+  } = community ?? {};
+
+  // Find contract information on community
+  const contract =
+    commnunityStrategies?.find((st) => st.name === proposalStrategy)
+      ?.contract ?? {};
+
+  const {
+    name: tokenName,
+    maxWeight: maxWeightFromContract,
+    minBancale: minBalancefromContract,
+  } = contract;
+
+  // get maxWeight and minBalance from proposal or default to community settings
+  const maxWeight = proposalMaxWeight ?? maxWeightFromContract;
+  const minBalance = proposalMinBalance ?? minBalancefromContract;
```

Comment:
making this to take max and min from proposal first and later from community contract